pkg/webhook/admission/http.go

```diff
@@ -97,5 +97,7 @@ func (wh *Webhook) writeResponse(w io.Writer, response Response) {
 	if err != nil {
 		wh.log.Error(err, "unable to encode the response")
 		wh.writeResponse(w, Errored(http.StatusInternalServerError, err))
+	} else {
+		wh.log.V(1).Info("encoded response", "admission response", responseAdmissionReview)
```

Comment:
responseAdmissionReview is not encoded.
```suggestions
wh.log.V(1).Info("admission response", "response", responseAdmissionReview)
```

Reply:
done